volatility/framework/plugins/windows/poolscanner.py

```diff
@@ -85,12 +85,22 @@ def __call__(self, data: bytes, data_offset: int):
                 if constraint.page_type is not None:
                     checks_pass = False
 
+                    # we only care about these types even though Windows supports a lot more pool types.
+                    # see also https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/wdm/ne-wdm-_pool_type
+                    # and Kernel Pool Exploitation on Windows 7 by Tarjei Mandt (2011))
+                    if header.PoolType > PoolType.FREE | PoolType.NONPAGED | PoolType.PAGED:
```

Comment:
needs verification because I think this might be wrong. @iMHLv2 @ikelos

Reply:
resolved because outdated. While I am still unsure about this, I removed it for now because after more testing it doesn't appear necessary.